dell/src/main/java/org/apache/iceberg/dell/EcsClient.java

```diff
@@ -0,0 +1,226 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.iceberg.dell;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Function;
+import org.apache.iceberg.catalog.Namespace;
+import org.apache.iceberg.catalog.TableIdentifier;
+import org.apache.iceberg.exceptions.NoSuchNamespaceException;
+
+/**
+ * ECS client and its util-methods.
+ */
+public interface EcsClient extends AutoCloseable {
+
+  /**
+   * utils of object key that provide methods to convert {@link ObjectKey} to other objects declared in Iceberg.
+   *
+   * @return utils class
+   */
+  ObjectKeys getKeys();
+
+  /**
+   * utils of properties that provide methods to convert object to properties.
+   *
+   * @return utils class
+   */
+  default PropertiesSerDes getPropertiesSerDes() {
```

Comment:
The Iceberg community doesn't use `get` in method names unless something needs to conform to Java Bean expectations. The reason is that `get` is almost never valuable: either there is a more specific verb that should be used instead, or it is basically useless and should be omitted.

Reply:
Fixed!